packages/cf-component-card/src/CardToolbarLink.js

```diff
@@ -3,25 +3,27 @@ import PropTypes from 'prop-types';
 import Link from 'cf-component-link';
 import { createComponent } from 'cf-style-container';
 
-const styles = ({ isActive, theme }) => ({
+const styles = ({ isActive, expandable, theme }) => ({
   position: 'relative',
   display: 'inline-block',
   padding: '0.73333rem 2em 0.73333rem 0.5rem',
   lineHeight: theme.lineHeight,
 
-  '&::after': {
-    content: "'\\f003'",
-    fontFamily: '"cloudflare-font"',
-    fontStyle: 'normal',
-    fontWeight: 'normal',
-    marginTop: '-0.7em',
-
-    position: 'absolute',
-    top: '51%',
-    right: '1rem',
-    transition: 'transform 0.2s ease',
-    transform: isActive ? 'rotate(90deg)' : 'initial'
-  }
+  '&::after': expandable
```

Comment:
Could we use the `Icon` component here?

Reply:
It's not so obvious, (I would say it's impossible) because of css animation we use to expande/collapse motion.

Reply:
I'm going make this a common component soon. I'll swap out the places that use it. We have rotating link one offs everywhere.

Reply:
@manatarms Yep, this is such a common pattern now. That would be ace!